IBPSA/Fluid/Movers/Preconfigured/FlowControlled_dp.mo

```diff
@@ -0,0 +1,39 @@
+within IBPSA.Fluid.Movers.Preconfigured;
+model FlowControlled_dp "Fan or pump with ideally controlled head dp as input signal and pre-configured parameters"
+  extends IBPSA.Fluid.Movers.FlowControlled_dp(
+    final per(
+            pressure(
+              V_flow=m_flow_nominal/rho_default*{0, 1, 2},
+              dp=if rho_default < 500
+                   then dp_nominal*{1.12, 1, 0}
+                   else dp_nominal*{1.14, 1, 0.42}),
+            powerOrEfficiencyIsHydraulic=true,
+            etaHydMet=IBPSA.Fluid.Movers.BaseClasses.Types.HydraulicEfficiencyMethod.EulerNumber,
+            etaMotMet=IBPSA.Fluid.Movers.BaseClasses.Types.MotorEfficiencyMethod.GenericCurve),
+    final constantHead,
+    final heads,
+    final nominalValuesDefineDefaultPressureCurve=true,
+    final inputType=IBPSA.Fluid.Types.InputType.Continuous,
+    final prescribeSystemPressure=false,
+    final init=Modelica.Blocks.Types.Init.InitialOutput,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial);
+    annotation (
+defaultComponentName="mov",
+Documentation(info="<html>
+<p>
+This model is the preconfigured version for
+<a href=\"Modelica://IBPSA.Fluid.Movers.FlowControlled_dp\">
+IBPSA.Fluid.Movers.FlowControlled_dp</a>.
+</html>", revisions="<html>
+<ul>
+<li>
+August 17, 2022, by Hongxiang Fu:<br/>
+First implementation. This is for
+<a href=\"https://github.com/lbl-srg/modelica-buildings/issues/2668\">#2668</a>.
+</li>
+</ul>
+</html>"), Icon(graphics={Line(
+          points={{-100,60},{-80,40},{-40,80}},
```

Comment:
I suggest to delete this check mark as it makes then schematic diagram busy (and other pre-configured records don't have it either).

Reply:
Do we somehow differentiate the preconfigured models and the vanilla models at all? Their class names are the same and can be confusing.
Some models on Buildings side (e.g. `Buildings.Examples.ScalableBenchmarks.BuildingVAV.Examples.OneFloor_OneZone`) use both the two versions. (This is because only the fan/pump models using a dummy two-point pressure curve were swapped with the preconfigured version.)

Reply:
I deleted the check mark in b7fba88.
I tried to think of some other way to do the preconfigured model icon, but feel probably not worth the effort.